tox.ini

```diff
@@ -1,12 +1,14 @@
 [tox]
 skipsdist = True
-envlist = py37-lint, py37-mypy
+envlist = py{34,35,36,37}-unit, py37-lint, py37-mypy
```

Comment:
If there is going to be multiple tox files, is the idea to have a different build per folder? I think we need to define what makes sense in the project structure to "build differently".

Reply:
Still TBD, I don't have an opinion on this.